text_dedup/minhash.py

```diff
@@ -119,17 +110,14 @@ def embed_func(
         bytes(" ".join(t).lower(), "utf-8") for t in ngrams(NON_ALPHA.split(content), ngram_size, min_length)
     }
 
-    hashvalues: np.ndarray = np.array([hash_func(token) for token in tokens], dtype=dtype)
+    hashvalues: np.ndarray = np.array([hash_func(token) for token in tokens], dtype=dtype).reshape(len(tokens), 1)
```

Comment:
I think you can do this `np.array([[hash_func(token)] for token in tokens], dtype=dtype)`

Reply:
You could. but In my opinion, doing so :
Does not make the intended dimension change clear, 
Adds more python based list comprehensions which i would like to avoid due to potential difficulties that may arise directly or during porting to other code.

Reply:
I am okay with the first point and reshape being used here. But I have to say no to the second one: dumbing down the Pythonic-ness of code to make conversion to other languages easier is not the priority nor the goal.